gulp-ng-config.js

```diff
@@ -30,10 +32,24 @@ function gulpNgConfig (moduleName, configuration) {
         jsonObj,
         wrapTemplate;
 
-    try {
-      jsonObj = JSON.parse(file.contents.toString('utf8'));
-    } catch (e) {
-      this.emit('error', new PluginError(PLUGIN_NAME, 'invaild JSON file provided'));
+    if (_.endsWith(file.path, 'yml') || _.endsWith(file.path, 'yaml')) {
+      configuration.parser = 'yml';
```

Comment:
Pushed the change. Removed the default parser and handled it in the stream handler.